cmd/clusterctl/client/cluster/workload_cluster.go

```diff
@@ -21,12 +21,14 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	utilkubeconfig "sigs.k8s.io/cluster-api/util/kubeconfig"
+	utilsecret "sigs.k8s.io/cluster-api/util/secret"
 )
 
 // WorkloadCluster has methods for fetching kubeconfig of workload cluster from management cluster.
 type WorkloadCluster interface {
 	// GetKubeconfig returns the kubeconfig of the workload cluster.
 	GetKubeconfig(workloadClusterName string, namespace string) (string, error)
+	GetUserKubeconfig(workloadClusterName string, namespace string) (string, error)
```

Comment:
```suggestion

    // GetUserKubeconfig returns the system kubeconfig of the  workload cluster.
	GetUserKubeconfig(workloadClusterName string, namespace string) (string, error)
```

Reply:
"GetUserKubeconfig returns the system" doesn't sound right